src/QuickMarcEditor/QuickMarcEditor.js

```diff
@@ -31,41 +43,53 @@ const QuickMarcEditor = ({ instance, onClose, handleSubmit, initialValues, submi
       pristine={pristine}
     />
   ), [onClose, handleSubmit, submitting, pristine]);
-
   const initialRecords = initialValues?.records;
 
   useEffect(() => {
     if (initialRecords) setRecords(initialRecords);
   }, [initialRecords]);
 
+  const addRecord = useCallback(({ values }) => {
```

Comment:
let's call it `changeRecords` as it will be used in remove + use `!==` instead of `>`